programmers/난이도별/level01.문자열_내_p와_y의_개수/yoojinhee03.java

```diff
@@ -0,0 +1,24 @@
+package programmers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class yoojinhee03 {
+
+	public static void main(String[] args) {
+		int[] arr= {1,2,3,4,5};
+		System.out.println(solution("pPyyY"));
+		
+	}
+	
+	static boolean solution(String s) {
+
+        s=s.toUpperCase();
+        int cnt=0;
+        String arr[]=s.split("");

```

Comment:
```suggestion
        int cnt=0;
        String arr[] = s.toUpperCase().split("");
```
- 매개변수에 다시 값을 할당하는 것은 굉장히 위험한 행위입니다.
- 그냥 한 번에 uppercse => split 으로 표현해도 될 것 같아요!